ch-covidcertificate-backend-delivery/ch-covidcertificate-backend-delivery-ws/src/main/resources/application.properties

```diff
@@ -15,6 +15,8 @@ server.error.whitelabel.enabled=true
 
 jeap.logging.logrelay.host=logrelaycaasp.bit.admin.ch
 
+db.retentionPeriod=P30D
```

Comment:
remove this. default is enough. this only causes confusion as this .properties file is applied for the deployment, but can also be overwritten in the manifest.yaml
```suggestion
```